apis/v1alpha1/httproute_types.go

```diff
@@ -174,16 +174,23 @@ type HTTPRouteRule struct {
 	// Conformance-levels at this level are defined based on the type of filter:
 	// - ALL core filters MUST be supported by all implementations.
 	// - Implementers are encouraged to support extended filters.
-	// - Implementation-specific custom filters have no API guarantees across implementations.
+	// - Implementation-specific custom filters have no API guarantees across
+	//   implementations.
+	//
 	// Specifying a core filter multiple times has unspecified or custom conformance.
 	//
+	// If unspecified, no filters will be applied to matched requests.
+	//
 	// Support: core
 	//
 	// +optional
 	// +kubebuilder:validation:MaxItems=16
 	Filters []HTTPRouteFilter `json:"filters,omitempty"`
 
 	// ForwardTo defines the backend(s) where matching requests should be sent.
+	// If unspecified, the rule performs no forwarding. If unspecified and filters
+	// is unspecified, a 503 error code is returned.
```

Comment:
I think this needs to be clarified. We'd still return a 503 if filters were specified that didn't actually result in a response being sent.